cypress/integration/changeSignalAfterSubmit.spec.js

```diff
@@ -0,0 +1,324 @@
+// <reference types="Cypress" />
+import * as createSignal from '../support/commandsCreateSignal';
+import { CREATE_SIGNAL } from '../support/selectorsCreateSignal';
+import { CHANGE_CATEGORY, CHANGE_LOCATION, CHANGE_STATUS, CHANGE_TYPE, CHANGE_URGENCY, SIGNAL_DETAILS } from '../support/selectorsSignalDetails';
+import { MANAGE_SIGNALS, OVERVIEW_MAP } from '../support/selectorsManageIncidents';
+describe('Change signal after submit', () => {
+  describe('Create signal graffitie', () => {
+    before(() => {
+      cy.server();
+      cy.defineGeoSearchRoutes();
+      cy.getAddressRoute();
+
+      cy.visitFetch('incident/beschrijf');
+    });
+
+    it('Should search for an address', () => {
+    // Search on address
```

Comment:
Not blocking but I would add 2 space before '//' so your comments are consistent, I think I have an ESLint rule for this, so feel free to ignore, will fix this when I will add the rule ;)